tests/requirements.txt

```
@@ -12,7 +12,7 @@ menuinst >=2
 packaging
 pkginfo
 psutil
-py-lief
+py-lief <0.15
```

Removing this change since #5565 merged.
```suggestion
py-lief
```